src/main/python/run_trecqa.py

```diff
@@ -150,6 +161,7 @@ def get_precision(actual, predicted, k):
   parser.add_argument("-k", help="top-k passages", default=10)
   parser.add_argument("-model", help="[idf|sm]", default='idf')
   parser.add_argument('-index', help="path of the index", required=True)
+  parser.add_argument('--qa-model-file', help="the path to the model file")
```

Comment:
@gauravbaruah yes it's true that `--` is used for switches. But, we have been using single `-` with default values for optional arguments. To be consistent, I think we should use `-` and use default values maybe?